packages/maskbook/src/web3/UI/EthereumAccountButton.tsx

```diff
@@ -68,18 +68,16 @@ export function EthereumAccountButton(props: EthereumAccountButtonProps) {
     const selectedWallet = useWallet()
     const selectedWalletProvider = useValueRef(currentSelectedWalletProviderSettings)
 
-    const [, setSelectWalletDialogOpen] = useRemoteControlledDialog(WalletMessages.events.walletStatusDialogUpdated)
-    const [, setSelectProviderDialogOpen] = useRemoteControlledDialog(WalletMessages.events.selectProviderDialogUpdated)
+    const { openDialog: openSelectWalletDialog } = useRemoteControlledDialog(
+        WalletMessages.events.walletStatusDialogUpdated,
+    )
+    const { openDialog: openSelectProviderDialog } = useRemoteControlledDialog(
+        WalletMessages.events.selectProviderDialogUpdated,
+    )
     const onOpen = useCallback(() => {
-        if (selectedWallet)
-            setSelectWalletDialogOpen({
-                open: true,
-            })
-        else
-            setSelectProviderDialogOpen({
-                open: true,
-            })
-    }, [selectedWallet, setSelectWalletDialogOpen, setSelectProviderDialogOpen])
+        if (selectedWallet) openSelectWalletDialog()
+        else openSelectProviderDialog()
+    }, [selectedWallet, openSelectWalletDialog, openSelectProviderDialog])
```

Comment:
There is error, you can't replace the `setSelectProviderDialogOpen` to openDialog. the `setSelectProviderDialogOpen` is a alias name for `onUpdateLocal`, so here we should use `setDialog`. 

The `setDialog` and `openDialog` are two different behaviors. `setDialog` will call the `ev.sendLocal`.

Perhaps you have many other such incorrect substituions in your code, plz check theme.

Reply:
The error cause the `connect wallet` button to be unresponsive when clicked